First part of my particling madness

Donut05 opened the pull request.

Donut05 committed:
Campfire done

Donut05 committed:
Fixed a typo in documentation + adjustments

Donut05 committed:
Fixed all campfire related particles!

Donut05 committed:
Implemented "gas stove" particles for furnaces

Donut05 committed:
Funne pollution cloud

Donut05 committed:
Science particle 100% done

Donut05 committed:
Pollution particle + science particle adjustments

Donut05 committed:
Spline shenanigans

Donut05 committed:
Merge branch 'refactoring-madness' into donut/particling-madness

Donut05 committed:
Random upgrader particle

Donut05 committed:
Wow I organise stuff!

Donut05 committed:
Merge branch 'main' into donut/particling-madness

Donut05 committed:
I just want to  get this bs out of my way

Donut05 committed:
Basic updgrader particle done

Donut05 committed:
Fuck you tofix and red boi

Donut05 committed:
Merge branch 'main' into donut/particling-madness

Donut05 committed:
Merge branch 'main' into donut/particling-madness

Donut05 committed:
Update icon map

Donut05 committed:
Ee-eao-oo-eeeee-Ee-eao-oo-eeeee-aeou-oioi-aaa-uuuu

Donut05 committed:
Merge branch 'donut/particling-madness' of https://github.com/Dr-Pixel-Dev/Scrap-Factory into donut/particling-madness

Donut05 committed:
New basic upgrader particle, final

Donut05 committed:
Merge branch 'main' into donut/particling-madness

Donut05 committed:
Sorting effects

nunya-beezneez reviewed (dismissed):
I'd like you to update the message on commit `fcf836b`. I haven't reviewed much else, so this review is incomplete. I'll add a review request so somebody else can have a look on important parts.

Donut05 committed:
Desparately trying to reverse fucking description

Donut05 committed:
Revert "Desparately trying to reverse fucking description"

This reverts commit 8fd5b0479a61fc9be6ac48f9d44e00535b38ac52.

Donut05 committed:
Attempt #2

Donut05 committed:
Fixed upgrade particle rotation

Donut05 committed:
Merge branch 'main' into donut/particling-madness

Donut05 committed:
Merge branch 'main' into donut/particling-madness

Natejoestev committed:
Merge branch 'main' into donut/particling-madness

Natejoestev committed:
fix unclosed if statement

Natejoestev committed:
Merge branch 'donut/particling-madness' of https://github.com/Mel-On-E/Scrap-Factory into donut/particling-madness

Natejoestev committed:
fix indenting

Natejoestev committed:
fix indenting 2

Natejoestev committed:
fix indent #3

Donut05 committed:
Unfinished idle particle for the science furnace

Donut05 committed:
Merge branch 'donut/particling-madness' of https://github.com/Mel-On-E/Scrap-Factory into donut/particling-madness

Donut05 committed:
Merge branch 'main' into donut/particling-madness

Donut05 committed:
Description.json is now the same as the steam desc

I am pushing this cuz this more is dead and I don't like the blue ping on my github

Donut05 committed:
fuck

Donut05 committed:
Merge branch 'main' into donut/particling-madness

Donut05 committed:
Update description.json
